src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -189,18 +189,6 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
     ];
 
     this.serviceColumns = [
-      {
-        name: $localize`Service Name`,
-        prop: 'service_name',
-        flexGrow: 2,
-        filterable: true
-      },
-      {
-        name: $localize`Service Type`,
-        prop: 'service_type',
-        flexGrow: 1,
-        filterable: true
-      },
```

Comment:
We can avoid using the using the listTpl rather we can use the each field to present data.

Reply:
why removing these ?

Reply:
@cloudbehl didn't get you here

Reply:
@afreen23 , we show service events for each service in a tab named Service Events. Since the service name and service type are already available in the service details, so repeating this info here was redundant, hence decided to remove it and just keep the events column.

Reply:
Can you update the screenshot then, seems like not updated so could not follow, thanks

Reply:
Why removing this, because those fields are not adding value.

Reply:
Ack @aaSharma14 @cloudbehl